charts/gateway-helm/templates/envoy-proxy-topology-mutator-deployment.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if .Values.topologyWebhook.enabled }}
+apiVersion: apps/v1
```

Comment:
is it possible to reuse the EG deployment, and have 1 less moving part from an ops perspective

Reply:
https://github.com/envoyproxy/gateway/blob/c2075cd01f849269d7afd2a53654f8ab67e68ec8/internal/provider/kubernetes/kubernetes.go#L45

Reply:
Definitely, will update with that